app/components/grids.js

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image"
+
+export default function TravelCard({ imageUrl, priceColor, fill }) {
+  return (
+    <div className="rounded-lg bg-white shadow-sm transition-shadow hover:shadow-md">
+      <div className="relative">
+        <Image
+          alt="Photos of works"
+          height={200}
+          src={imageUrl}
+          width={400}
+        />
+        <div
+          className={`absolute left-4 top-4 rounded-full px-3 py-1 text-xs text-white font-roboto font-black tracking-eightLetter ${priceColor}`} aria-label="blog price $49/MO"
+        >
+          $49/MO
+        </div>
+      </div>
+      <div className="p-6">
+        <dl>
+          <dt className="text-lightBlack font-roboto mb-2 text-xl font-black">
+            Keeping the dream alive by traveling the world.
```

Comment:
Igual, estos textos se supone que deben venir de propiedades
```suggestion
            {title}
```